spec/implementations/pto/fetch_pto_from_drive_spec.rb

```
@@ -0,0 +1,63 @@
+
+    fake_service = instance_double(Google::Apis::SheetsV4::SheetsService)
+    fake_response = instance_double(Google::Apis::SheetsV4::ValueRange, values: [
+        ['ignored', 'Juan', 'ignored', '06/24/2025', '06/25/2025', '', '', 'PTO', '', 'active']
+    ])
+
+    allow(Google::Apis::SheetsV4::SheetsService).to receive(:new).and_return(fake_service)
+    allow(fake_service).to receive(:authorization=)
+    allow(fake_service).to receive(:get_spreadsheet_values).and_return(fake_response)
+
+    allow(mocked_shared_storage_reader).to receive(:read).and_return(
+        instance_double(Bas::SharedStorage::Types::Read, id: 1, data: { key: 'value' }, inserted_at: Time.now)
```

RuboCop (1.75.5)

[convention] Indent the first argument one step more than the start of the previous line.

(Layout/FirstArgumentIndentation)